update to 1.7.1

sannykr opened the pull request.

sannykr committed:
update to 1.7.1

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

sannykr committed:
Update meta.yaml

sannykr committed:
added the hardcoded link

I have added the hardcoded link to the pypi file because the default one was not working.

sannykr commented:
@synapticarbors : In this meta.yaml file, I used the hardcoded url to the pypi because the default one didn't work. Is it acceptable?

isuruf commented on recipe/meta.yaml:
```diff
@@ -1,13 +1,14 @@
 
 source:
     fn: django-redis-cache-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/d/django-redis-cache/django-redis-cache-{{ version }}.tar.gz
```
@sannykr, this is outdated, see https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

sannykr replied:
@isuruf : I don't understand what you are saying. I had to update the package on conda-forge. When I changed the version number and the md5, I got an error about pypi url. That's why I have used the hardcoded url. I was just asking if it is acceptable to use the hardcoded url of the package.

ocefpaf replied:
@sannykr please read the example that @isuruf pasted above.

sannykr replied:
@isuruf : Thanks for letting me know. I have made the required changes.

sannykr replied:
@isuruf : Doesn't it work for windows? I don't know why it is disabled. 
When I looked at the pypi page, I found that it is OS independent. Should I use noarch: python in the build section?

sannykr committed:
Update meta.yaml

sannykr committed:
removed the skip statement

I have removed the line that skips the windows build and want to see if it works for windows.

sannykr commented:
@kwilcox: Doesn't it work for windows?

Commit:
MNT: Re-rendered with conda-smithy 2.4.0

sannykr commented:
@isuruf: Could you please review and merge.

isuruf commented:
I don't have permission. Please ping the maintainers

kwilcox commented:
@sannykr As with the other PR, please rebase on top of master with (2) commits... the version bumpy and the re-render.

sannykr commented:
@kwilcox : I have opened another pull request with only two commits. Please let me know if that works.
